### DISCUSSION TITLE
Why does it load 53 mods

### DISCUSSION THREAD

**Discussion by Nikev2** [Sinytra/Connector#466] [Q&A] [answered]
Seriously i have 8 mods and forge says i have 53 mods loaded. Is it going to use up all 8gb of my ram?

**Comment by ghost** [comment, marked as answer]
The other mods are the fabric API being ported over to forge, they dont really count. Also, no, 8 mods shouldnt use 8 gb of your ram unless, somehow, said mods are that resource intensive.
